intel_extension_for_transformers/transformers/utils/config.py

```diff
@@ -987,7 +987,7 @@ def __init__(
         self.use_neural_speed = use_neural_speed
         self.device = kwargs.get("device", "auto")
         self.calib_dataloader = kwargs.get("calib_dataloader", None)
-        self.calib_len = kwargs.get("calib_len", None)

```

Comment:
will this impact the accuracy? @changwangss @VincyZhang  pls check

Reply:
if 2048 is needed for certain model, we should set in the example script, instead of the common config.

Reply:
2048 is example default value in auto-round project. Will confirm it.